sections/debt/components/ManageTab/index.tsx

```diff
@@ -1,12 +1,34 @@
+import Wei from '@synthetixio/wei';
+import useUserStakingData from 'hooks/useUserStakingData';
+import { useRecoilValue } from 'recoil';
+import { walletAddressState } from 'store/wallet';
 import styled from 'styled-components';
-
 import { FlexDivColCentered } from 'styles/common';
 
 const ManageTab = () => {
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { debtBalance: actualDebt } = useUserStakingData(walletAddress);
+
+	const uniswapURLWithQueryParams = (debt: Wei) =>
```

Comment:
No biggie, but some ideas for small improvements.
- Name this function `getDehedgeUniswapUrl` so give some better context.
- store the address in a const, something like `dHedgeTokenAddress`
- Can also move this function out of the render method

Reply:
The token is currently unknown, waiting for the dhedge team to port the token from polygon. 
For the other two points: yes, will change

Reply:
Yep cool!